### ISSUE TITLE
Tile Design

### ISSUE THREAD

**Issue by Choco31415** [OpenSourceTagpro/engine#1]
Should we make tiles modular? As in each gets their own class? If they do, what should their design be? What should they be able to read, and what should they be able to influence? Discuss below.

**Comment by sallaben** [comment]
Yes, they should definitely be modular. We need to figure out what each tile has in common - abstract it away to a base model.

**Comment by sallaben** [comment]
So I did some thinking. Each tile has this in common:

- some shape (that fits within a square)
- some effect on a ball that touches it
- some texture

Are there any other shared characteristics?
